ForkEat/ForkEat.Core/Repositories/ILikeRepository.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Threading.Tasks;
+
+namespace ForkEat.Core.Repositories
+{
+    public interface ILikeRepository
+    {
+        Task<bool> LikeRecipe(Guid userId, Guid recipeId);
+        Task UnlikeRecipe(Guid userId, Guid recipeId);
+        Task<bool> GetLike(Guid userId, Guid recipeId);
+    }
+}
```

Comment:
I don't understand the need for a ``LikeRepository` why not update the instance field on the recipe and just save the recipe using the existing repository method ?